deployment/generate-attributions.sh

```diff
@@ -0,0 +1,12 @@
+# generate or update the THIRD-PARTY-NOTICES
+# requires oss-attribution-generator - https://www.npmjs.com/package/oss-attribution-generator
+# appends the generated list to the base file that specifies rush attributions
+# must be run from the deployment/ directory where this resides
+
+cat ./third-party-notices-base.txt > ../THIRD-PARTY-NOTICES.txt
```

Comment:
i'm confused how this works. Does THIRD-PARTY-NOTICES just grow monotonically forever?

Reply:
No, this first step overwrites it with the base, then the subsequent step appends to it. It's a new file every time. But I couldn't figure out how this base was derived, so I'm not clearing it and instead only adding to it (the base is what was in THIRD-PARTY-NOTICES.txt before this change).

Reply:
ah you're right. I confused single ">" with ">>" (to append)